release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1277,6 +1277,47 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
+You are encouraged to upgrade your dynamic plugins to React Router v6 with the `react-router-dom-v5-compat` module. (link:https://issues.redhat.com/browse/OCPBUGS-34538[*OCPBUGS-34538*])
+
+* Previously, one inactive or idle browser tab caused session expiration for all other tabs. With this change, activity in any tab will prevent session expiration even if there is one inactive or idle browser tab. (link:https://issues.redhat.com/browse/OCPBUGS-34387[*OCPBUGS-34387*])
+
+* Previously, when the `Display Admission Webhook` warning implementation presented issues with some incorrect code. With this release, the unnecessary warning message has been removed. (link:https://issues.redhat.com/browse/OCPBUGS-35940[*OCPBUGS-34316*])
+
+* Previously, select `textareas` were not resizable. With this update, you are now able to resize `textareas`. (link:https://issues.redhat.com/browse/OCPBUGS-34200[*OCPBUGS-34200*])
+
+* Previously, the console Operator was not able to tolerate the absence of the ingress capability. With this update, the console Operator configuration API has been enhanced with the possibility to add alternative ingress for the environments where the ingress cluster capability is disabled. (link:https://issues.redhat.com/browse/OCPBUGS-33787[OCPBUGS-33787*])
+
+* Previously, the *Debug container* link was not displayed for pods with a `Completed` status. With this change, the link now appears. (link:https://issues.redhat.com/browse/OCPBUGS-33631[*OCPBUGS-33631*])
+
+* Previously, the {product-title} web console did not show `filesystem` metrics on the *Nodes list* page due to incorrect prom query. With this update, `filesystem` metrics are correctly displayed. (link:https://issues.redhat.com/browse/OCPBUGS-33136[*OCPBUGS-33136*])
+
+* Previously, pseudolocalization was not working due to a configuration issue. After this fix, pseudolocalization works again. (link:https://issues.redhat.com/browse/OCPBUGS-30218[*OCPBUGS-30218*])
+
+* Previously, console pods would crash loop if the `--user-auth` flag was set to `disabled`. With this update, the console backend properly handles this value. (link:https://issues.redhat.com/browse/OCPBUGS-29510[*OCPBUGS-29510*])
+
+* Previously, utilization cards displayed a `limit` incorrectly implying a relationship between capacity & limits. With this update, the position of `limit` was changed and the wording updated. (link:https://issues.redhat.com/browse/OCPBUGS-23332[*OCPBUGS-23332*])
+
+* Previously, in some edge cases, the wrong resource could be fetched when using websockets to watch a namespaced resource without providing a namespace. With this update, a validation to the resource watch logic was added to prevent the websocket request, and log an error under this condition. (link:https://issues.redhat.com/browse/OCPBUGS-19855[*OCPBUGS-19855*])
```

Comment:
```suggestion
* Previously, in some edge cases, the wrong resource could be fetched when using websockets to watch a namespaced resource without providing a namespace. With this update, a validation to the resource watch logic was added to prevent the websocket request and log an error under this condition. (link:https://issues.redhat.com/browse/OCPBUGS-19855[*OCPBUGS-19855*])
```